src/web/styles/themes.scss

```diff
@@ -30,8 +30,10 @@
   --theme-warning-background: #fefcd3;
   --theme-info-icon: #7085d4;
   --theme-error-icon: #e52900;
+  --theme-error-border: #e52900;
   --theme-success-icon: #035959;
   --theme-warning-icon: #ffdb1d;
+  --theme-warning-border: #ffdb1d;
```

Comment:
as above, maybe we don't need this and can just use a more generic `--theme-warning`

Reply:
changed